webextensions/api/windows.json

```diff
@@ -780,7 +780,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": "≤61"
+                    "version_added": "5"
```

Comment:
✅ https://github.com/chromium/chromium/commit/b9911c28e52d7d269adc2542b6198c21252bac98 in [5.0.349.0](https://github.com/chromium/chromium/releases/tag/5.0.349.0) before 5.0.375.